src/metalperformanceshaders.cs

```diff
@@ -174,10 +214,14 @@ interface MPSImageHistogram {
 		void EncodeToCommandBuffer (IMTLCommandBuffer commandBuffer, IMTLTexture source, IMTLBuffer histogram, nuint histogramOffset);
 
 		[Export ("histogramSizeForSourceFormat:")]
-		nuint HistogramSizeForSourceFormat (MTLPixelFormat sourceFormat);		
+		nuint HistogramSizeForSourceFormat (MTLPixelFormat sourceFormat);
+
+		//FIXME: Vector @property (readwrite, nonatomic) vector_float4 minPixelThresholdValue;
+		//[Export ("minPixelThresholdValue", ArgumentSemantic.Assign)]
+		//[unsupported ExtVector: float __attribute__((ext_vector_type(4)))] MinPixelThresholdValue { get; set; }
```

Comment:
`vector_float4` is OpenTK.Vector4. This will also need a `[MarshalDirective]` attribute.